.github/workflows/check.yaml

```
@@ -82,8 +82,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        runs-on: [[self-hosted, jammy, X64, large]]
-        test-command: ['tox -e func']
+        runs-on: [[self-hosted, jammy, X64, large], [Ubuntu_ARM64_4C_16G_01]]
+        test-command: ['if [ "$(uname -m)" = "aarch64" ]; then echo "skipping func tests on arm64"; else echo tox -e func; fi']
```

```suggestion
        test-command: ['if [ "$(uname -m)" = "aarch64" ]; then echo "skipping func tests on arm64"; else tox -e func; fi']
```

The logic looks like it's working, but there's a stray echo. :)